SwiftificationTests/SequenceTypeTests.swift

```diff
@@ -39,6 +39,27 @@ class SequenceTypeTests: XCTestCase {
         let actual = array.takeFirst { $0 > 2 }
         XCTAssertEqual(expected, actual)
     }
+    
+    func testTakeWhile1() {
```

Comment:
These new tests are cool, but none of them really test the changes to 'uniqueBy', unless I'm missing something. Maybe one more test that is a condition that will throw so you can make sure it handles that correctly?